### DISCUSSION TITLE
PHP: How to do slash pages to create side pages

### DISCUSSION THREAD

**Discussion by AustinGitHub** [community/community#21438] [Other Feature Feedback, Questions, & Ideas]
<p>Hi,<br>
How do I do pages like this<br>
<a href="http://site.com/Artist/Lil_Wayne" class="onebox" target="_blank" rel="noopener nofollow ugc">site.com/Artist/Lil_Wayne</a></p>
<p>So that Lil Wayne has it’s own page utilizing a code that I can repeat for any artist placed after the slash. I know in php I can do the question mark or hash parameters but I feel like there is a way to do it with the slash because sites like Genius, there is no way they have pages created for every single artist, it’s got to be utilizing some sort of repeating script so that they don’t have to recode everything for each artist. Example genius page - <a href="https://genius.com/artists/Kanye-west" rel="noopener nofollow ugc">https://genius.com/artists/Kanye-west</a> the kanye west isn’t using the slash or question mark parameter, so how can I do this but with my site?</p>
<p>Thanks,<br>
Austin</p>

**Comment by Zerotask** [comment, marked as answer]
<p>You can use a MVC framework like Laravel.</p>
